source/cli/appservices-push.txt

```diff
@@ -12,7 +12,7 @@ appservices push
    :depth: 1
    :class: singlecol
 
-Imports and deploys changes from your local directory to your app (alias: import)
+Pushes and deploys changes from your local directory to your app
```

Comment:
```suggestion
Imports and deploys changes from your local directory to your app (alias: import)
```
A little bit confusing, but the push command does technically 'import' local changes to your App